rmf_traffic_editor/gui/delete_dialog.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2019-2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef DELETE_DIALOG_H
+#define DELETE_DIALOG_H
+
+#include <QDialog>
+#include <QObject>
+#include <vector>
+#include <string>
+#include <utility>
+#include <map>
+#include "building.h"
+class QLineEdit;
+class QListWidget;
+class QLabel;
```

Comment:
There is a lot of unnecessary headers here (this should only contain what's needed in the header file, what's needed in the cpp can just go in there)

Also do we need to actually declare these QT classes as you are doing?

Reply:
Removed unused headers.